amplify/team-provider-info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dev": {
+    "awscloudformation": {
+      "AuthRoleName": "amplify-feedmobileapp-dev-230938-authRole",
```

Comment:
You do not want to commit the `amplify/*` folder. It will have stuff that cannot be pushed to a public repository